manifest/process.go

```diff
@@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
 		}
 	}
 
+	for i, p := range s.Ports {
+		// Assign random ports to "internal" balancers
+		if !p.Public {
+			p.Public = true
```

Comment:
Nope (as far as I’m aware), process is only used by start.
The fact that this is in the manifest package makes this a little confusion.

I’m a little confused as to the goal of the original issue now as private services should never collide anyway so will discuss this one with @mattmanning when he’s around



> On Aug 1, 2016, at 3:30 PM, David Dollar <notifications@github.com> wrote:
> 
> In manifest/process.go <https://github.com/convox/rack/pull/960#discussion_r72987170>:
> 
> > @@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
> >  		}
> >  	}
> >  
> > +	for i, p := range s.Ports {
> > +		// Assign random ports to "internal" balancers
> > +		if !p.Public {
> > +			p.Public = true
> Why do we flip this to public? Does this affect the creation of internal load balancers on AWS?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/convox/rack/pull/960/files/5214d5b69c13b6c2e0b44004d09f9029a828a197#r72987170>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA8uW8mxZXMEhg2opwyz-qeSv8RuJHycks5qbgL2gaJpZM4JZbmQ>.
>